backend/routes/calendar.routes.js

```diff
@@ -11,7 +11,8 @@ getAuthenticatedClient().then(auth => {
 
 router.post('/', async (req, res) => {
     try {
-        calendarApi.listEvents(req.body.start, req.body.end, req.body.count).then(events => res.send(JSON.stringify(events)));
```

Comment:
This should be a get route with query parameters, not post.
http://expressjs.com/ru/api.html#req.query

Also you can use res.json({ events }) instead of manually converting it.

Reply:
Fixed.